src/audit-log/audit-log.spec.ts

```diff
@@ -0,0 +1,72 @@
+import axios from 'axios';
+import MockAdapater from 'axios-mock-adapter';
+
+import WorkOS from '../workos';
+import { UnauthorizedException } from '../common/exceptions';
+
+const mock = new MockAdapater(axios);
+const event = {
+  group: 'WorkOS',
+  actor_name: 'WorkOS@example.com',
+  actor_id: 'user_1',
+  location: ' 192.0.0.8',
+  occurred_at: new Date(0),
+  target_name: 'Security Audit 2018',
+  target_id: 'document_39127',
+  action_type: 'U',
+  action: 'document.updated',
+};
+
+describe('AuditLog', () => {
+  describe('createEvent', () => {
+    describe('when the api responds with a 201 CREATED', () => {
+      it('posts Event successfuly', async () => {
```

Comment:
more a question than anything, but im not sure i follow what this is actually testing?

my naive reading is that we mock a 201, and then assert that there were no exceptions - in what world could this test fail?

Reply:
Ah good question. This test is in place for any future commits that may happen to break the really simple functionality.

Although it seems unlikely that this test would fail, I generally add something akin to this for each spec.